### ISSUE TITLE
Dependency Issues.

### ISSUE THREAD

**Issue by grabear** [vallenderlab/MicrobiomeR#84]
Unfortunately, the dplyr problems associated with #79 and #82 are causing more local problems.  In an attempt to resolve these issues I've started to use packrat.  I'm not sure if this will work or change my current problem, but I wanted to try.

Current error during build (`devtools::check()`):
```r
E  creating vignettes (1m 0.9s)
   The following 234 of 1955 (12%) input indexes have `NA` in their classifications:
      28, 29, 38, 39, 52, 56 ... 1938, 1941, 1943, 1944, 1948
   
   Your object is in the phyloseq format!
   Please format your Taxmap object to continue analysis.
   Your object is in the phyloseq format!
   Please format your Taxmap object to continue analysis.
   Ordered Data.
   Converted to the raw format.
   Ordered Data.
   Warning: There is no "taxon_id" column in the data set "3", so there are no taxon IDs.
   Warning: The data set "4" is named, but not named by taxon ids.
   Warning: There is no "taxon_id" column in the data set "3", so there are no taxon IDs.
   Warning: The data set "4" is named, but not named by taxon ids.
   Quitting from lines 48-96 (analysis.Rmd) 
   Error: processing vignette 'analysis.Rmd' failed with diagnostics:
   use of NULL environment is defunct
   Execution halted
Error in processx::run(bin, args = real_cmdargs, stdout_line_callback = real_callback(stdout),  : 
  System command error
Calls: <Anonymous> ... eval -> eval -> with_envvar -> force -> <Anonymous>
Execution halted

Exited with status 1.
```

**Comment by grabear** [comment]
In order to get Bioconductor packages properly installed remember to do:
`options(repos = c(getOption("repos"), BiocInstaller::biocinstallRepos()))`

**Comment by grabear** [comment]
@sdhutchins started at 4:59 PM, and then I was rooting around during installation so it errored (which deletes everything :frowning_face:)......
![image](https://user-images.githubusercontent.com/20953530/53454395-9cf6dd00-39ec-11e9-810a-e658f3712e7d.png)

**Comment by sdhutchins** [comment]
> In order to get Bioconductor packages properly installed remember to do:
> `options(repos = c(getOption("repos"), BiocInstaller::biocinstallRepos()))`

Aren't they trying to get people to use BiocManager now?

**Comment by grabear** [comment]
Yea, but that call just puts (`BiocInstaller::biocinstallRepos()`) simply prints a vector of urls that are used for downloading bioconductor packages.

**Comment by sdhutchins** [comment]
> Yea, but that call just puts (`BiocInstaller::biocinstallRepos()`) simply prints a vector of urls that are used for downloading bioconductor packages.

Is that all they were doing with that package? lol smh.

**Comment by sdhutchins** [comment]
Fixed in 93fcce6800b31c8ebdb335f8a3b504563825e08d. We still should probably add `dplyr (>=0.8.0)` to the DESCRIPTION file.

**Comment by sdhutchins** [comment]
Description file updated in #88.
